yapapi/runner/__init__.py

```diff
@@ -153,8 +153,13 @@ class _BufferItem(NamedTuple):
 
 
 class Engine(AsyncContextManager):
-    """Requestor engine. Used to run tasks based on a common package on providers."""
+    """
+    Requestor engine.
 
+    Used to run tasks using the specified application package within providers' execution units.
+    """
```

Comment:
Let's also add a module docstring. Something along the lines of "(An) implementation of the Requestor engine".

Reply:
@azawlocki done